__snapshots__/setup-cfg.js

```diff
@@ -24,7 +24,7 @@ author = Google LLC
 author_email = googleapis-packages@google.com
 
 license = Apache 2.0
-platforms = Posix, MacOS X, Windows
+platforms = POSIX, MacOS X, Windows
```

Comment:
This is the canonical and correct spelling: https://pypi.org/search/?c=Operating+System+%3A%3A+POSIX

Note that `MacOS X` is wrong, but it's messed up in PyPI, so I'm not proposing "fixing" it.